go/vt/sqlparser/ast_funcs.go

```diff
@@ -2257,6 +2257,18 @@ func AndExpressions(exprs ...Expr) Expr {
 // Equals is the default Comparator for AST expressions.
 var Equals = &Comparator{}
 
+// ToString returns the type as a string
+func (ty GeomFormatType) ToString() string {
+	switch ty {
+	case BinaryFormat:
+		return BinaryFormatStr
+	case TextFormat:
+		return TextFormatStr
+	default:
+		return "Unknown GeomFromWktType"
```

Comment:
This string seems incorrect. Maybe you copied the other function and forgot to change this string?